pkg/reconciler/openshift/openshift_test.go

```diff
@@ -64,15 +64,31 @@ func TestReconcileArgoCD_testDeployment(t *testing.T) {
 	assert.NilError(t, reconcilerHook(a, testDeployment))
 }
 
+func TestReconcileArgoCD_notInClusterConfigNamespaces(t *testing.T) {
+
+	a := makeTestArgoCD()
+	testClusterRole := &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      a.Name + "-" + testApplicationController,
+			Namespace: a.Namespace,
```

Comment:
The ClusterRole is cluster-scoped and not namespace-scoped.

Reply:
True ..